doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -230,17 +230,27 @@ We have successfully tested the following types:
 * MultiLineString
 * Polygon
 * MultiPolygon
-* Feature (using the value of `geometry` field at the first level)
 
-On the contrary, the following types doesn't work (you will get a "Database Error" if you try to use them):
+More information on the tests conducted can be found [here](https://github.com/telefonicaid/fiware-orion/issues/3586).
 
-* GeometryCollection
+The types `Feature` and `FeatureCollection` are also supported, but in a special way. You can
+use `Feature` or `FeatureCollection` to create/update `geo:json` attributes. However, when
+the attribute value is retrieved (GET resposes or notifictaions) you will get only the content of
 
-With regards to `FeatureCollection`, it is supported only if it contains a single Feature (i.e.
-the `features` field has only one element), in which case the value of `geometry` field of
-such element is used.
+* the `geometry` field, in the case of `Feature`
+* the `geometry` field of the first item of the `features` array, in the case of `FeatureCollection`
 
-More information on the tests conducted can be found [here](https://github.com/telefonicaid/fiware-orion/issues/3586).
+Note that actually Orion stores the full value used at `Feature` or `FeatureCollection`
+creation/updating time. However, from the point of view of normalization with other `geo:json` types,
+it has been decided to return only the `geometry` part. In the future, maybe a flag to return
+the full content would be implemented (more detail [in this issue](https://github.com/telefonicaid/fiware-orion/issues/4125)).
+
+Additionally, with regards to `FeatureCollection`, and addition check is done. It is accepted at
+creation/update time only if it contains a single `Feature` (i.e. the `features` field has
+only one element).
```

Comment:
```suggestion
With regards to `FeatureCollection`, it is only accepted at creation/update time only if it contains a single 
`Feature` (i.e. the `features` field has only one element). Otherwise , Orion would return an `BadRequest`error
with the following `description`: `features in FeatureCollection has more than 1 item`

```

Reply:
Fixed in [f6969d1](https://github.com/telefonicaid/fiware-orion/pull/4126/commits/f6969d13359241aeec562d09b1544d41ea946c01)

Reply:
In that make sense to delete the `description` explanation in the text. I still makes sense. 
```suggestion
With regards to `FeatureCollection`, it is only accepted at creation/update time only if it contains a single 
`Feature` (i.e. the `features` field has only one element). Otherwise , Orion would return an `BadRequest`error.

```